ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/ScanConfigsPage.tsx

```diff
@@ -25,34 +25,31 @@ function ScanConfigsPage() {
 
     return (
         <Switch>
-            <Route
-                exact
-                path={complianceEnhancedSchedulesPath}
-                render={() => {
-                    if (pageAction === 'create' && hasWriteAccessForCompliance) {
-                        return <CreateScanConfigPage />;
-                    }
-                    if (pageAction === undefined) {
-                        return (
-                            <ScanConfigsTablePage
-                                hasWriteAccessForCompliance={hasWriteAccessForCompliance}
-                            />
-                        );
-                    }
-                    return <Redirect to={complianceEnhancedSchedulesPath} />;
-                }}
-            />
-            <Route
-                exact
-                path={scanConfigDetailsPath}
-                render={() => {
-                    return (
-                        <ScanConfigDetailPage
-                            hasWriteAccessForCompliance={hasWriteAccessForCompliance}
-                        />
-                    );
-                }}
-            />
+            <Route exact path={complianceEnhancedSchedulesPath}>
+                <Route
+                    // eslint-disable-next-line react/no-children-prop
+                    children={() => {
+                        if (pageAction === 'create' && hasWriteAccessForCompliance) {
+                            return <CreateScanConfigPage />;
+                        }
+                        if (pageAction === undefined) {
+                            return (
+                                <ScanConfigsTablePage
+                                    hasWriteAccessForCompliance={hasWriteAccessForCompliance}
+                                />
+                            );
+                        }
+                        return <Redirect to={complianceEnhancedSchedulesPath} />;
+                    }}
+                />
+            </Route>
+            <Route exact path={scanConfigDetailsPath}>
+                {() => (
+                    <ScanConfigDetailPage
+                        hasWriteAccessForCompliance={hasWriteAccessForCompliance}
+                    />
+                )}
```

Comment:
This seems a little odd, providing an anonymous function as a child element instead of the JSX directly. Why is this needed versus rendering `<ScanConfigDetailPage>` as a direct child?